### PR TITLE
Update Windows Projects

### PR DESCRIPTION
**Pull request by NexeyaSGara**
Update all solutions for the new structure

### CODE REVIEW

File: win/PyTango_VS9/PyTango.vcproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="Windows-1252"?>
 <VisualStudioProject
 	ProjectType="Visual C++"
-	Version="9.00"
+	Version="9,00"
```
Comment (tiagocoutinho) [inline win/PyTango_VS9/PyTango.vcproj]: Is the comma a typo?

File: win/PyTango_VS9/PyTango.vsprops [hunk 2]
```diff
@@ -90,30 +90,30 @@
...
 
-    <UserMacro Name="PyTangoBaseFile" Value="_PyTango" />
+    <UserMacro Name="PyTangoBaseFile" Value="_tango" />
     
-    <UserMacro Name="TangoMajorVer" Value="8" />
-    <UserMacro Name="TangoMinorVer" Value="1" />
+    <UserMacro Name="TangoMajorVer" Value="9" />
+    <UserMacro Name="TangoMinorVer" Value="2" />
     <UserMacro Name="TangoPatchVer" Value="2" />
     <UserMacro Name="TangoVer" Value="$(TangoMajorVer).$(TangoMinorVer).$(TangoPatchVer)" />
```
Comment (tiagocoutinho) [inline win/PyTango_VS9/PyTango.vsprops]: Missing to change patch version to 0

File: win/PyTango_VS9/PyTango.vcproj (hunk 1 above)
Comment (NexeyaSGara) [inline win/PyTango_VS9/PyTango.vcproj]: In fact, both are working. Maybe it's my french version of Visual that changed it. So we can keep the .

File: win/PyTango_VS9/PyTango.vsprops (hunk 2 above)
Comment (NexeyaSGara) [inline win/PyTango_VS9/PyTango.vsprops]: the current Tango release for Windows is indeed 9.2.2, so maybe we have to update PyTango version to match it ?

Comment (vxgmichel) [inline win/PyTango_VS9/PyTango.vsprops]: @NexeyaSGara See what I wrote in the [readme](https://github.com/tango-cs/PyTango/tree/tango9#requirements):

> As a general rule, libtango and pytango should share the same major and minor version (for a version X.Y.Z, X and Y should match)

**Pull request merged**